server/src/routes.ts

```diff
@@ -904,6 +906,7 @@ export default async function routes(fastify: FastifyInstance) {
   fastify.get(
     '/events',
     {
+      preValidation: [fastify.optionalAuthenticate, ],
```

Comment:
This endpoint should not include any validation. API is open for the community and other services (such as Etherscan, OpenSea). We should not limit the response here

Reply:
One of the requirements was that comunity only can see comunity created events and admin is the only one that can see all (comunity + admin created). this is no longer necesary?

Reply:
Filter will always be made on the frontend app.
We should check on QR assignment (list or range) that the event assigned was created by the community (if the user is not admin)